pmemcheck/pmc_main.c

```diff
@@ -294,16 +339,33 @@ print_multiple_stores(void)
 static void
 print_store_stats(void)
 {
+/* print store states of the array */
+    struct pmem_st *tmp;
+    for (int s_index=0; s_index<=pInfo.info_array.m_index; s_index++) {
```

Comment:
make a temporary variable for `pInfo.info_array`. This will allow you to shorten this code quite a bit.